Dockerfile

```diff
@@ -2,9 +2,3 @@ FROM nginxinc/nginx-unprivileged:alpine
 
 COPY ./dist /app
 COPY config/nginx.conf /etc/nginx/conf.d/default.conf
-
-USER root
-
-RUN rm -rf /usr/share/nginx/html
```

Comment:
This is unnecessary, Docker has layers anyway, so it's not as if we were reducing the size of the image doing this, and we already replace the configuration where this is referenced from